SuperBuild/External_VTK.cmake

```diff
@@ -163,10 +138,7 @@ if((NOT DEFINED VTK_DIR OR NOT DEFINED VTK_SOURCE_DIR) AND NOT Slicer_USE_SYSTEM
     )
 
   set(_git_tag)
-  if("${Slicer_VTK_VERSION_MAJOR}" STREQUAL "8")
-    set(_git_tag "97904fdcc7e73446b3131f32eac9fc9781b23c2d") # slicer-v8.2.0-2018-10-02-74d9488523
-    set(vtk_egg_info_version "8.2.0")
-  elseif("${Slicer_VTK_VERSION_MAJOR}" STREQUAL "9")
+  if("${Slicer_VTK_VERSION_MAJOR}" STREQUAL "9")
```

Comment:
In the past, when removing support for older VTK, to ensure using git blame report the original motivation leading to the integration of specific code .... I kept the indent (with/without keeping the associated `if("${Slicer_VTK_VERSION_MAJOR}" STREQUAL "9")` like done here.